### DISCUSSION TITLE
(How) can I change the CSS transition used by the Progress Bar?

### DISCUSSION THREAD

**Discussion by gkruiger** [mantinedev/mantine#4526] [Q&A] [answered]
First of all: thanks for this amazing UI library.
It's extensive and easy to use and my main reason to develop a personal project in React, rather than using Svelte.

My question:
Animation of a change of value for the Progess Bar is handled by a CSS animation: `transition: width 100ms linear`.
Is there any way to change this?

**Comment by rtivital** [comment]
Thanks for kind words! You can change styles of any component with Styles API.

**Reply by gkruiger** [reply]
You deserve it.
Ok, I'll give that a try and come back with the results.

**Reply by gkruiger** [reply, marked as answer]
Works like a charm, for example:

```
<Progress 
  styles={{ 
    bar: { 
      transition: 'width 250ms ease'
    } 
  }}
/>
```

**Comment by BenSilver1337** [comment]
Thanks!
